### PR TITLE
Fix merge issues

### PR DESCRIPTION
**Pull request by nukappa**
WIP. Currently fixes issue #53

### CODE REVIEW

**Comment by nukappa** [comment]
Added documentation on the `merge_samples` command that clarifies its usages and closes #69

File: setup.cfg [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = spacemake
-version = 0.7.2
+version = 0.7.3
```
Comment (danilexn) [inline setup.cfg:3]: Now we have 0.7.4, so please change this accordingly.
When merging #64, this will be handled properly (automatically)

**Review by danilexn — COMMENTED** [review]
All changes look good. Please look at the comment I added for `setup.cfg` (updating version no.)

File: setup.cfg (hunk 1 above)
Comment (nukappa) [reply setup.cfg]: Thx, just bumped version and will merge the PR

**Pull request merged**